common/varnish.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Varnish Cache, a high-performance HTTP accelerator
+After=network-online.target
+
+[Service]
+Type=forking
+
+# Maximum number of open files (for ulimit -n)
+LimitNOFILE=131072
+
+# Locked shared memory - should suffice to lock the shared memory log
+# (varnishd -l argument)
+# Default log size is 80MB vsl + 1M vsm + header -> 82MB
+# unit is bytes
+LimitMEMLOCK=85983232
+
+# On systemd >= 228 enable this to avoid "fork failed" on reload.
+#TasksMax=infinity
```

Comment:
Is it a problem to enable it on systemd < 228?

Reply:
I'm not sure, and I have little prior knowledge about the original issue, I'm researching this now. For the record, we have these main targets:

```
debian-8-amd64
systemd 215

debian-9-amd64
systemd 232

centos-7-amd64
systemd 219

ubuntu-16.04-amd64
systemd 229
```

Reply:
Tested on debian jessie (systemd 215) and centos7 (systemd 219), which I believe to be the only distros in our targets that have systemd < 228. No immediate issues there. 
I think we should try adding this as a default.